README.md

```
@@ -40,6 +40,39 @@ uv tool install envoic      # Python
+envoic is available as an [Agent Skill](https://agentskills.io) for coding agents:
+
+| Tool | Auto-detected file | Notes |
+|---|---|---|
+| Cursor | `.cursorrules` | Generated from `skills/envoic/templates/cursor.cursorrules` |
+| GitHub Copilot | `.github/copilot-instructions.md` | Generated from `skills/envoic/templates/copilot-instructions.md` |
+| OpenAI Codex | `.agents/skills/envoic/SKILL.md` | Symlink/copy of canonical skill |
+| Claude Code | `.claude-plugin/plugins.yaml` | Generated from `skills/envoic/templates/claude-plugins.yaml` |
+
+Install from marketplace in Claude Code:
+
+    /plugin marketplace add mahimailabs/envoic
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)